libcudacxx/docs/extended_api/synchronization_primitives/barrier/memcpy_async_tx.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
This should go in the Asynchronous operations section next to `memcpy_async`, not in the barrier section.